canary/build.gradle

```diff
@@ -37,6 +37,9 @@ dependencies {
     // Prometheus
     implementation 'io.micrometer:micrometer-registry-prometheus:1.12.2'
 
+    // RocksDB
+    implementation 'org.rocksdb:rocksdbjni:9.0.0'
```

Comment:
Interesting, Kafka Streams is still on the 7.x.x I think. Tell Matthias to hurry up and upgrade!